modules/prometheus-exporter_oracle/conf/01-listener.yaml

```diff
@@ -0,0 +1,18 @@
+module: oracle
+name: "Process listener"
+id: listener
+
+transformation: true
+aggregation: true
+
+signals:
+  signal:
+    metric: tnslsnr
+
+rules:
+  critical:
+    threshold: 1
+    comparator: "<"
+    lasting_duration: "5m"
+    description: "is not started"
+tip: "the listener process is not started, means that application can not connect to the database"
```

Comment:
hopefully you set a tip or I could never understand that `tnslsnr` metric is related to a "listener process" !

this could be a stupid question but: if the listener process is not started, so application cannot connect to database just like for the prometheus exporter no? in this case this detector seems redundant with the previous one heartbeat.